tests/integration/lib/couch.rb

```diff
@@ -31,9 +31,10 @@ def prefix(db)
     "cozy" + Digest::SHA256.hexdigest(db)[0...32]
   end
 
-  def get_doc(domain, doctype, id)
+  def get_doc(domain, doctype, id, opts = {})
+    params = { params: opts }
```

Comment:
I'm curious: why this change?

Reply:
During my tests, I queried some docs with `meta: true` in order to see the conflicts as well as the rev history. It is not used in the final code, but I thought it might be useful in the future.